lib/classes/Variables.js

```diff
@@ -17,6 +17,7 @@ class Variables {
     this.envRefSyntax = RegExp(/^env:/g);
     this.optRefSyntax = RegExp(/^opt:/g);
     this.selfRefSyntax = RegExp(/^self:/g);
+    this.importsRefSyntax = RegExp(/^imports:/g);
```

Comment:
Unfortunately this is not possible.

The idea with this PR is that you define `imports` in your `serverless.yml` file. Those `imports` are then translated to `Fn::ImportValue` statements for AWS which are basically attached to the `imports` statement in the variable system.

The extension in the variable system only checks if the value is implicitly defined in the `imports` section so that users can't simply use the `imports` from the variable system if they are not defined in the `imports` section.

Using `self` here won't help since we're not populating anything.